dsls/sql/src/test/java/org/apache/beam/dsls/sql/planner/MockedBeamSqlTable.java

```diff
@@ -49,6 +48,7 @@
   public static final ConcurrentLinkedQueue<BeamSqlRow> CONTENT = new ConcurrentLinkedQueue<>();
 
   private List<BeamSqlRow> inputRecords;
+  private PCollection.IsBounded isBounded = PCollection.IsBounded.BOUNDED;
```

Comment:
not sure if this can test both `BOUNDED` and `UNBOUNDED` case, as `Create.of()` returns a BOUNDED PCollection.  Could use `TestStream` IMO.